styles/japanesewithanime.com/catppuccin.user.css

```diff
@@ -115,7 +117,9 @@
 	input.gsc-input{
 		color: @text !important;
 		background-color: @surface0 !important;
-		border: 4px solid @surface0 !important;
+		border-color: @surface0 !important;
+		border-width: 4px !important;
+    border-style: solid !important;
```

Comment:
Why are we setting anything other than `border-color`?

Reply:
i've removed some of the size changes i think are unnecessary but here are some reasons for the remaining ones:
- for `input.gsc-search-button` and `input.gsc-input` i feel like the original border style doesn't work for the theme and it looks odd without the slight rounding to replicate the rounding of the border and the size change to make the search button the same size as the search bar

- for `#archive-year-select` and `#archive-month-select` i feel like a border looks bad and slight rounding replicates the border look in the original theme

- for `.megaNav__Particles > a, .megaNav__Symbols > a` i also feel like a border looks bad but it looks weird without slight rounding

- for `.post-donation-cta` same as above